stories/accordion.stories.js

```diff
@@ -331,7 +344,6 @@ export const Default = () => {
   );
 };
 
-Default.story = {
-  name: 'default',
-  parameters: { notes: { markdown: notes } }
-};
+Default.args = { isOpen, size: defaultButtonSize, variant: variantOptions[0] };
```

Comment:
Setting `args` on the story replaces repetitive inline knob calls like `select('Variant', variantOptions)`.